nlu/spellbook.py

```diff
@@ -10284,8 +10285,14 @@ class Spellbook:
                                                   'de.resolve.icd10gm': 'sbertresolve_icd10gm',
                                                   'de.resolve.snomed': 'sbertresolve_snomed'},
                                               'en': {
+                                                  'en.text_generator.biomedical_biogpt_base': 'text_generator_biomedical_biogpt_base',
+                                                  'en.text_generator.generic_flan_base': 'text_generator_generic_flan_base',
+                                                  'en.text_generator.generic_jsl_base': 'text_generator_generic_jsl_base',
+                                                  'en.text_generator.generic_flan_t5_large': 'text_generator_generic_flan_t5_large',
+                                                  'en.text_generator.biogpt_chat_jsl': 'biogpt_chat_jsl',
+                                                  'en.text_generator.biogpt_chat_jsl_conversational': 'biogpt_chat_jsl_conversational',
+                                                  'en.text_generator.biogpt_chat_jsl_conditions': 'biogpt_chat_jsl_conditions',
```

Comment:
this should be `generate` instead of `text_generator` like in bart PR. Please update Docs/Tests accordigly, otherwise looks good!